server/src/main/java/org/elasticsearch/index/mapper/JsonFieldMapper.java

```diff
@@ -315,23 +315,23 @@ public Query rangeQuery(Object lowerTerm,
         @Override
         public Query fuzzyQuery(Object value, Fuzziness fuzziness, int prefixLength, int maxExpansions,
                                 boolean transpositions) {
-            throw new UnsupportedOperationException("[fuzzy] queries are not currently supported on [" +
-                CONTENT_TYPE + "] fields.");
+            throw new UnsupportedOperationException("[fuzzy] queries are not currently supported on keyed " +
+                "[" + CONTENT_TYPE + "] fields.");
```

Comment:
I don't know what is your plan on this but it should be possible to allow this type of query by adding the field prefix in the fuzzy query and change the prefixLength to be `prefixLength + field.length + 1` ?